requirements.txt

```diff
@@ -26,19 +26,19 @@ pytest-repeat==0.9.4
 pytest-github-actions-annotate-failures==0.3.0
 pytest==8.4.1
 recommonmark==0.7.1
-regex==2024.11.6
-ruff==0.12.1
-schemdraw==0.20
-scikit-learn==1.7.0
+regex==2025.7.34
```

Comment:
Didn't look at the release notes for this yet, but mainly wondering why we even have this package added? It's just an expanded regex module so I don't expect a ton of issues.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/2426 - it was very helpfully merged with zero description! This is always why I push hard for good PR descriptions 🙂